styles/main.css

```diff
@@ -1,7 +1,991 @@
 html {
     box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+    font-family: 'Roboto', sans-serif;
 }
 
-  *, *:before, *:after {
+
+*, *:before, *:after {
     box-sizing: inherit;
-}
\ No newline at end of file
+    margin: 0;
+    padding: 0;
+}
+
+a {
+    text-decoration: none;
+}
+
+ul {
+    list-style-type: none;
+}
+
+main {
+    margin: 0.1em 0.5em;
+}
+
+section:nth-child(n+2):nth-last-child(n+1) {
+    max-width: 1194px;
+    margin: auto;
+}
+
+.flex {
+    display: flex;
+}
+
+.wrap {
+    flex-wrap: wrap;
+}
+
+.col {
+    flex-direction: column;
+}
+.jc-sb {
+    justify-content: space-between;
+}
+
+.jc-c {
+    justify-content: center;
+}
+
+.jc-sa {
+    justify-content: space-around;
+}
+
+.c-pointer {
+    cursor: pointer;
+}
+
+/**** START Navigation ****/
+
+ul {
```

Comment:
Careful with duplicate selectors. You first used this one on line 19

Reply:
Thanks for catching this one, resolved on commit https://github.com/adriaanbd/techy-media-site/pull/14/commits/cae50dc45ad218c11cfa49f78c3497c62d235c4e